tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "skipLibCheck": true,
     "target": "ESNext",
     "module": "ESNext",
-    "moduleResolution": "Node",
+    "moduleResolution": "Bundler",
```

Comment:
Is this relevant? https://x.com/mattpocockuk/status/1775138087443595689

Reply:
Kind of yes, it was because `node` does not resolve types for `@nuxt/eslint-config/flat` - while I can fix it on the plugin side, I think it's more future-proof to change to Bundler. I could send another PR to do it